docs/DeveloperGuide.md

```diff
@@ -958,6 +1033,42 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Updating sales of drinks
+
+1. Update the sales of a drink item while all sales are being shown.
```

Comment:
From the delete example given in the DG, should this line be "updating the sales of a drink...?"

Reply:
I think I was unclear because I put test cases for updating one drink and more than one drinks under the first one. I will separate them.